radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendMessage.groovy

```diff
@@ -0,0 +1,35 @@
+package com.radixdlt.cli
+
+import com.radixdlt.client.application.RadixApplicationAPI
+import com.radixdlt.client.atommodel.accounts.RadixAddress
+import org.radix.utils.RadixConstants
+import picocli.CommandLine
+
+@CommandLine.Command(name = "send-message", mixinStandardHelpOptions = true,
+        description = "Send Message")
+class SendMessage implements Runnable {
+
+    @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+    Composite.IdentityInfo identityInfo
+
+    @CommandLine.Option(names = ["-m", "--message"], paramLabel = "MESSAGE", description = "message to send", required = true)
+    String messageString
+
+    @CommandLine.Option(names = ["-d", "--address"], paramLabel = "ADDRESS", description = "Address to which message is sent", required = true)
+    String addressString
+
+    void run() {
+
+        RadixApplicationAPI api = Utils.getAPI(identityInfo)
+        RadixAddress address = RadixAddress.from(addressString)
+
+        println "Sending message '${messageString}' to address ${address}"
+        RadixApplicationAPI.Result result = api.sendMessage(address, messageString.getBytes(RadixConstants.STANDARD_CHARSET), true)
+        result.blockUntilComplete()
+        println "Message sent successfully using AtomID : ${result.getAtom().getAid()}"
```

Comment:
_"using AtomID"_ sounds strange.. maybe: _"Message sent successfully, aid of resulting atom:"
 or similar?

Reply:
done